CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
```

Comment:
Hi. Should we add something here relating to the fact that you have added the datetime frequency interval parameter?

Reply:
That's a valid remark, I added it to the changelog.